nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -64,7 +63,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
             seed=self.cfg.get('seed', 1234),
         )
 
-        if not self.cfg.get('fused_bf16'):
+        if self.cfg.get('precision') != 'bf16':
```

Comment:
I don't think we have access to the train config here right? Isn't this the model config that doesn't have the precision param?

Reply:
We add it to the config in the training script.